test/unit/udp_test.c

```diff
@@ -29,7 +29,8 @@ int main(int argc, char ** argv)
     buffer target = vector_pop(unassoc);
     u32 daddr;
     u16 dport;
-    parse_v4_address_and_port(target, &daddr, &dport);
+    if (!parse_v4_address_and_port(target, &daddr, &dport))
+        test_perror("parse");
```

Comment:
test_perror() prints the string representation of the `errno` variable value, which is not set when a parsing error occurs. We should use test_error() instead, and the error string should be more informative, e.g. "failed to parse address and port".

Reply:
fixed!